packages/wpcom-proxy-request/src/index.ts

```diff
@@ -268,31 +300,29 @@ function patchFileObjects( formData ) {
 	//   so it's detectable by the `supportsFileConstructor` code.
 	// - `window.chrome` exists also on Edge (!), `window.chrome.webstore` is only in Chrome and
 	//   not in other Chromium based browsers (which have the site isolation bug, too).
-	if ( ! window.chrome || ! supportsFileConstructor ) {
+	if ( ! ( window as { chrome?: unknown } ).chrome || ! supportsFileConstructor ) {
 		return;
 	}
 
 	for ( let i = 0; i < formData.length; i++ ) {
 		const val = getFileValue( formData[ i ][ 1 ] );
 		if ( val ) {
-			formData[ i ][ 1 ] = new window.File( [ val ], val.name, { type: val.type } );
+			formData[ i ][ 1 ] = new window.File( [ val ], ( val as File ).name, {
+				type: ( val as File ).type,
+			} );
 		}
 	}
 }
 
 /**
- * Injects the proxy <iframe> instance in the <body> of the current
- * HTML page.
+ * Injects or reloads the proxy <iframe> instance in the <body> of the current HTML page.
  */
-
-function install() {
+function install(): void {
 	debug( 'install()' );
 	if ( iframe ) {
 		uninstall();
 	}
 
-	buffered = [];
-
```

Comment:
Why this change?

Reply:
Buffered was a nullable array, now it's just an array. The `uninstall` function handles clearing the array if necessary. It's no longer necessary to set it to an empty array for initialization.